cloudbuild.yaml

```
@@ -0,0 +1,15 @@
+  args: ['build', '-t', '$_GCR/$_PROJ_NAME/$_REPO_NAME/$_IMG_NAME', '.']
+# Push the container image to Container Registry
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['push', '$_GCR/$_PROJ_NAME/$_REPO_NAME/$_IMG_NAME']
+# Deploy container image to Cloud Run
+- name: 'gcr.io/google.com/cloudsdktool/cloud-sdk'
+  entrypoint: 'gcloud'
+  args: ['run', 'deploy', '$_REPO_NAME',
+         '--image', '$_GCR/$_PROJ_NAME/$_REPO_NAME/$_IMG_NAME',
+         '--region', '$_REGION',
+         '--memory', '$_MEMORY',
+         '--cpu', '$_CPU',]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)